### ISSUE TITLE
model.eval()

### ISSUE THREAD

**Issue by Daibao1120** [truantmuse/chinese-qa-with-bert#13]
請問如何得知eval loss?
